Assets/Scripts/LevelSelection.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class LevelSelection : MonoBehaviour
+{
+    public Button[] lvlButtons;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        int levelAt = PlayerPrefs.GetInt("levelAt", 2);
+
+        for (int i = 0; i < lvlButtons.Length; i++)
+        {
+            if ((i + 2) > levelAt)
```

Comment:
There is no - 1 after lvlButtons.Length in the current code, I'm confused. I'm fairly certain the code is i + 2 because the lvlButtons array has the lvls starting at index 0 while the build order .GetInt("levelAt") would have the lvls starting at index 2 due to main menu being 0 and level selection being 1.